purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -462,6 +466,40 @@ class Purchases internal constructor(
         purchasesOrchestrator.getCustomerInfo(fetchPolicy, true, callback)
     }
 
+    /**
+     * Fetches the virtual currencies for the current subscriber.
+     *
+     * @param callback A listener called when the virtual currencies are available.
+     */
+    fun getVirtualCurrencies(
+        callback: GetVirtualCurrenciesCallback,
+    ) {
+        virtualCurrencyManager.virtualCurrencies(callback = callback)
+    }
+
+    /**
+     * Invalidates the cache for virtual currencies.
+     *
+     * This is useful for cases where a virtual currency's balance might have been updated
+     * outside of the app, like if you decreased a user's balance from the user spending a virtual currency,
+     * or if you increased the balance from your backend using the server APIs.
+     *
+     * For more info, see our [virtual currency docs](https://www.revenuecat.com/docs/offerings/virtual-currency)
+     */
+    fun invalidateVirtualCurrenciesCache() {
+        virtualCurrencyManager.invalidateVirtualCurrenciesCache()
+    }
+
+    /**
+     * The currently cached [VirtualCurrencies] if one is available.
+     * This is synchronous, and therefore useful for contexts where an app needs a [VirtualCurrencies]
+     * right away without waiting for a callback.
+     *
+     * This allows initializing state to ensure that UI can be loaded from the very first frame.
+     */
```

Comment:
Would be worth adding a note that this will remain empty until virtual currencies have been fetched at least once using `getVirtualCurrencies`?